packages/teams-js/src/public/marketplace.ts

```diff
@@ -0,0 +1,361 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { validateCartItems, validateCartStatus, validateUuid } from '../internal/marketplaceUtils';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * @hidden
+ * Namespace for an app to support a checkout flow by interacting with the marketplace cart in the host.
+ * @beta
+ */
+export namespace marketplace {
+  /**
+   * @hidden
+   * Represents the cart object for the app checkout flow.
+   * @beta
+   */
+  export interface Cart {
+    /**
+     * @hidden
+     * Version of the cart.
+     */
+    readonly version: CartVersion;
```

Comment:
Were I to be a developer looking at this property and documentation, I don't think I would understand what this property is for. I know what a version is but... what should I be using it for?

Reply:
Makes sense to me! I would suggest you update the docs just to make it clear to developers that they can just ignore this field (e.g., just add something like "this is for internal use only")